presto-main/src/main/java/com/facebook/presto/operator/ConnectorCommitStrategy.java

```diff
@@ -0,0 +1,7 @@
+package com.facebook.presto.operator;
+
+public enum ConnectorCommitStrategy
```

Comment:
what about unifying the name to `PageSinkCommitStrategy`? @arhimondr , @shixuan-fan